src/standalone-js/webpack/build.config.js

```diff
@@ -0,0 +1,23 @@
+const {resolve} = require('path');
```

Comment:
Nit: personally prefer single well organized webpack files with env switching rather than directories of webpack configs that include each other.